docs/introduction/index.md

```diff
@@ -13,9 +13,9 @@ support.
 
 * [Zoned Storage Overview](./zoned-storage.md): Principle of operation of zoned storage devices.
 
-* [Shingled Magnetic Recording](./smr.md): Learn about Shingled Magnetic
+* [SMR HDDs](./smr.md): Learn about Shingled Magnetic
```

Comment:
I would prefer "Shingled Magnetic Recording Hard-Disks". But I do not think adding hard-disks buys us much here.

Reply:
In this section, we're talking about Zoned Storage, associated specifications and their implementations. Having the title as just 'Shingled Magnetic Recording' does not directly connect to zoned storage and may be confusing for a newbie trying to understand the overall picture of this section. IMO, using the word 'hard-disks' will be helpful for a newbie to easily connect and get an overview of the sub sections.

An alternate more explanatory title for this sub-section could be 'ZAC/ZBC compliant SMR Hard-Disks'. This will align well with the other sub-section title 'NVMe Zoned Namespaces compliant SSDs'